src/transactions.ts

```diff
@@ -51,3 +51,10 @@ export interface Transaction {
   installments?: TransactionInstallments;
   category?: string;
 }
+
+export interface FutureDebit {
```

Comment:
since this is not part of the transactions, it shouldn't be here. you can move them to `src/scrapers/base-scraper.ts` right above `ScaperScrapingResult` interface

Reply:
done